### ISSUE TITLE
Stop token at triggered tilie

### ISSUE THREAD

**Issue by dmaneyapanda** [1000nettles/hey-wait#5]
Ideally if a player drags a token past a trigger tile, the movement should stop at that tile and pause the game, rather than continuing to it's original end point past the triggering tile before pausing.

**Comment by 1000nettles** [comment]
Agreed, this is a bit of a heavier lift but is planned for the future.
